src/aliceVision/hdr/hdr_test_common.hpp

```diff
@@ -0,0 +1,176 @@
+#pragma once
+#include <aliceVision/image/all.hpp>
+#include <aliceVision/system/Logger.hpp>
+
+#include "rgbCurve.hpp"
+#include "sampling.hpp"
+
+#include <boost/filesystem.hpp>
+#include <random>
+#include <array>
+
+using namespace aliceVision;
```

Comment:
Avoid "using namespace" in headers (even if I agree that's a particular case for unit tests).
Maybe good to make it explicit that these functions are for tests using a namespace or functions name (currently it's only in the filename).

Reply:
Agreed. Fixed now.